From 1f98359c654a8f7b393f367db8ca509c0e01f303 Mon Sep 17 00:00:00 2001
From: Divya Ranjan <divya@subvertising.org>
Date: Wed, 11 Dec 2024 01:03:46 +0000
Subject: [PATCH 03/10] gnu: Add rust-graphene-sys-0.15.

* gnu/packages/crates-gtk.scm (rust-graphene-sys-0.15): New variable.

Change-Id: I438f63d806fdcbb8e07d75388be6db09faa68963
---
 gnu/packages/crates-gtk.scm | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/gnu/packages/crates-gtk.scm b/gnu/packages/crates-gtk.scm
index 73acd5e3c2..e5a52555d9 100644
--- a/gnu/packages/crates-gtk.scm
+++ b/gnu/packages/crates-gtk.scm
@@ -2959,6 +2959,32 @@ (define-public rust-graphene-sys-0.18
        #:cargo-development-inputs (("rust-shell-words" ,rust-shell-words-1)
                                    ("rust-tempfile" ,rust-tempfile-3))))))
 
+(define-public rust-graphene-sys-0.15
+  (package
+    (name "rust-graphene-sys")
+    (version "0.15.10")
+    (source
+      (origin
+        (method url-fetch)
+        (uri (crate-uri "graphene-sys" version))
+        (file-name
+          (string-append name "-" version ".tar.gz"))
+        (sha256
+          (base32
+	    "12h2qcdhvzxhkc75fqkky6rz212wp2yc6mgvk9cxz8bv6g3iysgs"))))
+    (build-system cargo-build-system)
+    (arguments
+      `(#:cargo-inputs
+        (("rust-glib-sys" ,rust-glib-sys-0.15)
+         ("rust-libc" ,rust-libc-0.2)
+         ("rust-pkg-config" ,rust-pkg-config-0.3)
+         ("rust-system-deps" ,rust-system-deps-6))))
+    (home-page "https://gtk-rs.org/")
+    (synopsis "FFI bindings to libgraphene-1.0")
+    (description
+      "This package provides FFI bindings to libgraphene-1.0.")
+    (license license:expat)))
+
 (define-public rust-gsk4-0.8
   (package
     (name "rust-gsk4")
-- 
2.46.0

